scripts/nginx/radarr.sh

```diff
@@ -36,7 +36,7 @@ cat > /home/"$user"/.config/Radarr/config.xml << RADARR
   <UpdateMechanism>BuiltIn</UpdateMechanism>
   <BindAddress>127.0.0.1</BindAddress>
   <Port>7878</Port>
-  <SslPort>8787</SslPort>
+  <SslPort>9898</SslPort>
```

Comment:
Changes like these are the ones that will require updater scripts. Those should probably come in a different, isolated PR so we can test that procedure independently from the radarr installer.